src/test/kotlin/no/nav/syfo/mocks/AzureAdMock.kt

```diff
@@ -0,0 +1,22 @@
+package no.nav.syfo.mocks
+
+import io.ktor.client.engine.mock.*
+import io.ktor.client.request.*
+import io.ktor.client.request.forms.*
+import io.ktor.http.*
+import no.nav.syfo.client.azuread.AzureAdTokenResponse
+
+private fun azureAdTokenResponse(token: String?) = AzureAdTokenResponse(
+    access_token = token ?: "token",
+    expires_in = 3600,
+    token_type = "type",
+)
+
+fun MockRequestHandleScope.getAzureAdResponse(request: HttpRequestData): HttpResponseData {
+    val token = (request.body as FormDataContent).formData["assertion"]
+    return respond(
```

Comment:
Forsøkt meg på mock-engine i isaktivitetskrav og da lagde jeg en hjelpe-metode for dette (inspirert av et annet repo) - kan jo være noe å vurdere. Ref: https://github.com/navikt/isaktivitetskrav/commit/e2352ca0aa8f83d662ba15e50b515b0cf4d7223c#diff-3ba5087c4613f6b1082fdd6676330d4e74a7456dfd4b96424036f728dda8a80eR17

Reply:
Det er ikke alltid vi returnerer samme status for samme url, feks når vi mocker at tjenesten feiler. Så da gikk vinninga opp i spinninga..